novawallet/Modules/Staking/StakingType/StakingTypeInteractor.swift

```diff
@@ -66,9 +67,15 @@ extension StakingTypeInteractor: StakingTypeInteractorInputProtocol {
             $0.delegate = self
             $0.start()
         }
+
+        if case .direct = stakingSelectionMethod.selectedStakingOption {
```

Comment:
Don't really want to have staking specific logic in the universal class. That's why we have made recommendation logic so complicated in terms of architecture. Can you, please, highlight a use case for that?